File: deck/seed/code/native/kotlin/runtime/locale.kt
// Locale runtime. Reached only through the public environment API.
object tongue {
    fun tag(): String = java.util.Locale.getDefault().toLanguageTag()

    fun timezone(): String = java.util.TimeZone.getDefault().id

    fun preferred(): MutableList<String> = mutableListOf(tag())
}

File: deck/seed/code/native/node/runtime/locale.ts
// Locale runtime for node. `Intl` is a JavaScript built-in rather than a node module, so it is reached here rather
// than through a `dock load` of a node namespace. Reached only through the public environment API.
const locale = {
  options: (): Intl.ResolvedDateTimeFormatOptions =>
    Intl.DateTimeFormat().resolvedOptions(),
}
